build_tools/github/build_wheels.sh

```diff
@@ -8,11 +8,23 @@ if [[ "$RUNNER_OS" == "macOS" ]]; then
     # Make sure to use a libomp version binary compatible with the oldest
     # supported version of the macos SDK as libomp will be vendored into the
     # scikit-learn wheels for macos. The list of binaries are in
-    # https://packages.macports.org/libomp/.  Currently, the oldest
-    # supported macos version is: High Sierra / 10.13. When upgrading this, be
-    # sure to update the MACOSX_DEPLOYMENT_TARGET environment variable in
-    # wheels.yml accordingly. Note that Darwin_17 == High Sierra / 10.13.
-    wget https://packages.macports.org/libomp/libomp-11.0.1_0+universal.darwin_17.i386-x86_64.tbz2 -O libomp.tbz2
+    # https://packages.macports.org/libomp/.
+    if [[ "$CIBW_BUILD" == *-macosx_arm64 ]]; then
+        # arm64 builds must cross compile because CI is on x64
+        export PYTHON_CROSSENV=1
+        # SciPy requires 12.0 on arm to prevent kernel panics
+        # https://github.com/scipy/scipy/issues/14688
+        # We use the same deployment target to match SciPy.
+        export MACOSX_DEPLOYMENT_TARGET=12.0
+        wget https://packages.macports.org/libomp/libomp-11.0.1_0.darwin_20.arm64.tbz2 -O libomp.tbz2
+    else
+        # Currently, the oldest supported macos version is: High Sierra / 10.13.
+        # When upgrading this, be sure to update the MACOSX_DEPLOYMENT_TARGET
+        # environment variable in wheels.yml accordingly. Note that Darwin_17 ==
+        # High Sierra / 10.13.
```

Comment:
I cannot find `MACOSX_DEPLOYMENT_TARGET` in `wheels.yml`. I suppose this comment refers to an older way of doing the CI config.

```suggestion
        # Note that Darwin_17 == High Sierra / 10.13.
```